nnpackage/examples/v1.3.0/two_circles/metadata/MANIFEST

```diff
@@ -0,0 +1,11 @@
+{
+  "major-version" : "1",
+  "minor-version" : "3",
```

Comment:
v1.1.0 - `configs`
v1.2.0 - `model_types`
v1.3.0 - `pkg-inputs`, `pkg-outputs`, and `model-connect`.